test/e2e/e2e.go

```diff
@@ -443,6 +443,33 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 	})
 })
 
+// Test pod connectivity to other host IP addresses
+var _ = ginkgo.Describe("test e2e pod connectivity to host addresses", func() {
+	const (
+		ovnWorkerNode    string = "ovn-worker"
+		targetIP         string = "123.123.123.123"
+		svcname          string = "node-e2e-to-host"
+	)
+
+	f := framework.NewDefaultFramework(svcname)
+
+	ginkgo.AfterEach(func() {
+		_, _ = runCommand("docker", "exec", ovnWorkerNode, "ip", "a", "del",
+			fmt.Sprintf("%s/32", targetIP), "dev", "eth0")
+	})
+
+	ginkgo.It("Should validate connectivity from a pod to a non-node host address on same node", func() {
+		// Add another IP address to the worker
+		_, err := runCommand("docker", "exec", ovnWorkerNode, "ip", "a", "add",
```

Comment:
Not sure if this should be expanded with test cases on `breth0`/other interfaces, so that we cover adding IP addresses on any interface?

Reply:
i should have used breth0 here. I don't think there are any other real interfaces in a kind deployment. I think we are good if we just cover this interface. We should also add ipv6 support, but I'll defer that until later.